### ISSUE TITLE
Session endpoint should return username

### ISSUE THREAD

**Issue by timwis** [CityOfPhiladelphia/restful-ben#17]
So that the UI can easily show the currently logged in user. We could alternatively implement a `/user` endpoint to get those details, but it would be convenient if `/session` returned it alongside `csrf_token` for easy local storage.

**Comment by awm33** [comment]
@timwis To start thinking in a more auth service word, I would hold off, since eventually you will need to hit a `/user` endpoint in auth service land

**Comment by timwis** [comment]
Makes sense. So we should create an issue about a `/user` endpoint then -- would that be in the restful-ben or taskflow repo?
